pss/pss.go

```diff
@@ -243,6 +319,7 @@ func (p *Pss) Start(srv *p2p.Server) error {
 
 func (p *Pss) Stop() error {
 	log.Info("Pss shutting down")
+	p.outbox.close()
```

Comment:
would that be a hack to just share the quitC with the outbox? then no  `close`   function is needed

Reply:
Same comment as above, I wanted to close the `process` channel to finish the main loop `for slot := range p.outbox.process {`

Reply:
Removed in 1b91b66